packages/app/aws-config/README.md

```diff
@@ -0,0 +1,2 @@
+# AWS config utils library
+TODO
```

Comment:
we need to document all supported env variables, but probably that should be a separate PR